ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -374,18 +374,19 @@ instance CardanoHardForkConstraints c
       ]
 
   supportedNodeToClientVersions _ = Map.fromList $
-      [ (NodeToClientV_1, CardanoNodeToClientVersion1)
-      , (NodeToClientV_2, CardanoNodeToClientVersion1)
-      , (NodeToClientV_3, CardanoNodeToClientVersion2)
-      , (NodeToClientV_4, CardanoNodeToClientVersion3)
-      , (NodeToClientV_5, CardanoNodeToClientVersion4)
-      , (NodeToClientV_6, CardanoNodeToClientVersion5)
-      , (NodeToClientV_7, CardanoNodeToClientVersion6)
-      , (NodeToClientV_8, CardanoNodeToClientVersion6)
-      , (NodeToClientV_9, CardanoNodeToClientVersion7)
+      [ (NodeToClientV_1 , CardanoNodeToClientVersion1)
+      , (NodeToClientV_2 , CardanoNodeToClientVersion1)
+      , (NodeToClientV_3 , CardanoNodeToClientVersion2)
+      , (NodeToClientV_4 , CardanoNodeToClientVersion3)
+      , (NodeToClientV_5 , CardanoNodeToClientVersion4)
+      , (NodeToClientV_6 , CardanoNodeToClientVersion5)
+      , (NodeToClientV_7 , CardanoNodeToClientVersion6)
+      , (NodeToClientV_8 , CardanoNodeToClientVersion6)
+      , (NodeToClientV_9 , CardanoNodeToClientVersion7)
+      , (NodeToClientV_10, CardanoNodeToClientVersion7)
       ]
 
-  latestReleasedNodeVersion _prx = (Just NodeToNodeV_7, Just NodeToClientV_9)
+  latestReleasedNodeVersion _prx = (Just NodeToNodeV_7, Just NodeToClientV_10)
```

Comment:
I'm thinking out loud here, in case someone sees something objectionable.

It's fine to bump this, since it's inconsequential whether the next release/release candidate tries to negotiate this new version. Esp: if they talk to a node that doesn't support it, they'll simply negotiate the eg previous version. Thus, as long as the `cardano-node` dependence on these new queries is well-guarded wrt to the negotiated version, then it's fine to "enable" them here.